tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn.py

```diff
@@ -71,26 +70,46 @@ def parameters(cls):
 
         return parameters
 
+    def is_detection_out(self):
+        if self.detection_out:
+            return True
+
+        return False
+
+    def is_box_outputs(self):
+        if self.classes_out and self.scores_out and self.boxes_out:
+            return True
+
+        return False
+
+    def is_num_detections_out(self):
```

Comment:
it is not what I meant.
1. The checks should be used before setting parameters or in the same place where they set
2. I am not sure that we need these methods at all in your context, we always can check property directly where it is necessary instead of methods.